File: backend/src/main/java/com/paintlearning/app/dto/response/FeedbackDTO.java
package com.paintlearning.app.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FeedbackDTO {
    private String id;
    private List<String> strengths;
    private List<String> weaknesses;
    private List<SuggestionDTO> suggestions;
    private Integer overallRating;
}

File: service/DTO/FeedbackDTO.cs
namespace PaintLearning.Service.DTO;

public class FeedbackDTO
{
    public string Id { get; set; } = string.Empty;
    public List<string> Strengths { get; set; } = new List<string>();
    public List<string> Weaknesses { get; set; } = new List<string>();
    public List<SuggestionDTO> Suggestions { get; set; } = new List<SuggestionDTO>();
    public int OverallRating { get; set; }
}

public class SuggestionDTO
{
    public string Area { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public string Priority { get; set; } = string.Empty;
    public bool IsResolved { get; set; }
}